fast64_internal/oot/oot_upgrade.py

```diff
@@ -0,0 +1,26 @@
+from bpy.types import Object, CollectionProperty
+from .data.oot_object_data import OoT_ObjectData
+
+
+def upgradeObjectList(objList: CollectionProperty, objData: OoT_ObjectData):
+    """Transition from the old object system to the new one"""
```

Comment:
"old" and "new" isn't specific enough, everything becomes "old" eventually
probably refer to the object-id-enum vs object-key-enum-from-xml instead